packages/types/src/types/ProvidersInventory.ts

```diff
@@ -92,7 +92,7 @@ export interface VSphereInventory {
   /** The self link of the entity. */
   selfLink: string;
   /** The type of the entity. */
-  type: 'openshift';
+  type: 'vsphere';
```

Comment:
wow, can you make a quick fix PR with the fixes in this file, so we can merge them now ?

Reply:
created https://github.com/kubev2v/forklift-console-plugin/pull/438